src/diskquota_utility.c

```diff
@@ -1235,12 +1236,11 @@ set_per_segment_quota(PG_FUNCTION_ARGS)
 int
 worker_spi_get_extension_version(int *major, int *minor)
 {
-	StartTransactionCommand();
-	int ret = SPI_connect();
-	Assert(ret = SPI_OK_CONNECT);
-	PushActiveSnapshot(GetTransactionSnapshot());
+	int state = 0;
```

Comment:
state initialization is redundant, the variable is initialized in SPI_connect_wrapper()

Reply:
In this place - yes, but in places with try/catch - no.